core/src/main/scala/org/apache/spark/ExecutorAllocationManager.scala

```diff
@@ -101,6 +101,9 @@ private[spark] class ExecutorAllocationManager(
   private val executorIdleTimeoutS = conf.getTimeAsSeconds(
     "spark.dynamicAllocation.executorIdleTimeout", "60s")
 
+  private val cachedExecutorTimeoutS = conf.getTimeAsSeconds(
```

Comment:
the variable name should match the config name to be consistent: `cachedExecutorIdleTimeoutS`